package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "react-supervenn",
+  "version": "0.0.1",
+  "description": "An interactive React implementation of [supervenn](https://github.com/gecko984/supervenn).",
+  "main": "dist/lib/index.js",
+  "module": "dist/lib/index.js",
+  "scripts": {
+    "start": "webpack serve -c demo.webpack.config.js --mode development",
+    "build:umd": "webpack -c webpack.config.js --mode production",
+    "build:esx": "NODE_ENV=production babel src/lib --out-dir dist/lib --copy-files",
+    "build:demo": "webpack -c demo.webpack.config.js --mode production",
+    "build": "npm run build:umd && npm run build:esx && npm run build:demo"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MaayanLab/react-supervenn.git"
+  },
+  "author": "MaayanLab",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/MaayanLab/react-supervenn/issues"
+  },
+  "homepage": "https://github.com/MaayanLab/react-supervenn#readme",
+  "devDependencies": {
+    "@babel/cli": "^7.16.8",
+    "@babel/plugin-transform-runtime": "^7.14.5",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-dynamic-import-webpack": "^1.1.0",
+    "babel-plugin-remove-webpack": "^1.1.0",
+    "css-loader": "^5.2.6",
+    "file-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.1",
+    "mini-css-extract-plugin": "^1.6.0",
+    "react-dom": "^17.0.2",
+    "style-loader": "^2.0.0",
+    "terser-webpack-plugin": "^5.1.3",
+    "url-loader": "^4.1.1",
+    "use-async-resource": "^2.2.1",
+    "webpack": "^5.39.0",
+    "webpack-cli": "^4.7.2",
+    "webpack-dev-server": "^3.11.2"
+  },
+  "dependencies": {
+    "@babel/core": "^7.17.5",
```

Comment:
I don't think this should be in dependencies, perhaps it belongs in dev dependencies if it's required